class RunnerError(Exception):
    pass


class Runner:
    def run(self):
        pass
